systemtest/060-delete.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+#
+# Copy tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+
+    enable_delete=true
+    start_registry --enable-delete=$enable_delete reg
+}
+
+# delete image from registry
+@test "delete: remove image from registry" {
+    local remote_image=docker://busybox:latest
+    local localimg=docker://localhost:5000/busybox:unsigned
+    local output=
+
+    run_skopeo copy --dest-tls-verify=false $remote_image $localimg
+    output=$(run_skopeo inspect --tls-verify=false --raw $localimg)
+    echo $output | grep "vnd.docker.distribution.manifest.v2+json"
+
+    run_skopeo delete --tls-verify=false $localimg
```

Comment:
Non-blocking: Might be worth also checking that the image is really no longer accessible after deleting it.

Reply:
Yes, skopeo list command is a PRE feature now, I will use skopeo inspect command instead